app/components/Header/DropdownContents/ProfileDropdown/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import { Link, NavLink } from 'react-router-dom';
+import Icon from 'app/components/Icon';
+import styles from './ProfileDropdown.css';
+import shared from 'app/components/Header/Header.css';
+
+type Props = {
+  username: string,
+  logout: () => void,
+};
+
+const ProfileDropdown = ({ username, logout }: Props) => (
+  <div className={styles.profileDropdownEl}>
+    <div className={shared.dropdownSection} data-first-dropdown-section>
+      <Link className={styles.username} to="/users/me">
+        {username}
+      </Link>
+      <NavLink
+        className={styles.dropdownLink}
+        activeClassName={styles.activeDropdownLink}
+        to="/users/me/settings/profile"
+      >
+        <Icon name="settings" prefix="ion-md-" size={20} />
+        <span>Innstillinger</span>
+      </NavLink>
+      <NavLink
+        className={styles.dropdownLink}
+        activeClassName={styles.activeDropdownLink}
+        to="/meetings"
+      >
+        <Icon name="calendar" size={20} />
+        <span>Møteinnkallinger</span>
+      </NavLink>
+    </div>
+    <div className={shared.dropdownSection}>
+      <span className={shared.bottomLink} onClick={logout}>
+        Logg ut
+        <Icon name="log-out" size={24} />
+      </span>
```

Comment:
I feel like this is weird being a link? and the navlinks above look more like buttons? I think some coherency with the other dropdown menus really won't hurt.

Reply:
This is also something I've thought a lot about, and thought I would wait and see what you guys thought. Interesting to see that you also caught onto this. Although its not really a "link", it does simultaneously route you to a "new" page, the front page.